### PR TITLE
One-off query support

### PR DESCRIPTION
**Pull request by kazimuth**
## Description of Changes
Adds an async one-off-query method to SpacetimeDBClient.

## API
Adds:
```cs
public async Task<T[]> SpacetimeDBClient.OneOffQuery<T>(string query) where T : IDatabaseTable;
```

## Requires SpacetimeDB PRs
https://github.com/clockworklabs/SpacetimeDB/pull/198

### CODE REVIEW

File: src/SpacetimeDBClient.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 using System.Reflection;
 using System.Text;
 using System.Threading;
+using System.Threading.Channels;
```
Comment (jdetter) [inline src/SpacetimeDBClient.cs:10]: I had to add a nuget package in order to get this working, did you not have to do this?

**Review by jdetter — APPROVED** [review]
Take a look at my comment before merging, but I was able to get this working.

File: src/SpacetimeDBClient.cs (hunk 1 above)
Comment (kazimuth) [reply src/SpacetimeDBClient.cs:10]: Not with `dotnet build`, but it's possible that pulled in some floating dependency from my system?

**Comment by jdetter** [comment]
Merging this into `staging`, this will go into `v0.7.0-beta`

**Pull request merged**